web/components/feed/feed-items.tsx

```diff
@@ -118,24 +118,80 @@ function FeedItem(props: { item: ActivityItem }) {
       return <FeedResolve {...item} />
     case 'commentInput':
       return <CommentInput {...item} />
+    case 'commentThread':
+      return <FeedCommentThread {...item} />
   }
 }
 
+export function FeedCommentThread(props: {
+  contract: Contract
+  comments: Comment[]
+  betsBySameUser: Bet[]
+  truncate?: boolean
+  smallAvatar?: boolean
+}) {
+  const { contract, comments, betsBySameUser, truncate, smallAvatar } = props
+  const [showCommentReplyInput, setShowCommentReplyInput] = useState(false)
+  const [replyToUsername, setReplyToUsername] = useState('')
+  const router = useRouter()
+  const scrollToId = '#' + comments[comments.length - 1].id
+  function scrollAndOpenReplyInput(comment: Comment) {
+    setReplyToUsername(comment.userUsername)
+    // We should replace this with refs to detect if the comment is already in view
+    router.replace(scrollToId, undefined, { shallow: true })
+    setShowCommentReplyInput(true)
+  }
+  return (
+    <div className={'w-full flex-col flex-col space-x-8 pr-6'}>
+      {comments.map((comment, commentIdx) => (
+        <div
+          key={comment.id}
+          id={comment.id}
+          className={clsx('flex space-x-3', commentIdx === 0 ? '' : 'mt-4')}
```

Comment:
Maybe add ml-8 and remove the space-x-8 above to be clearer how the indentation works